schemaregistry/schemaregistry_client.go

```diff
@@ -672,9 +672,10 @@ func (c *Compatibility) ParseString(val string) error {
 
 // Fetch compatibility level currently configured for provided subject
 // Returns compatibility level string upon success
-func (c *client) GetCompatibility(subject string) (compatibility Compatibility, err error) {
+func (c *client) GetCompatibility(subject string, defaultToGlobal bool) (compatibility Compatibility, err error) {
 	var result compatibilityLevel
-	err = c.restService.handleRequest(newRequest("GET", subjectConfig, nil, url.PathEscape(subject)), &result)
+	err = c.restService.handleRequest(newRequest("GET", subjectConfigDefaultToGlobal, nil,
```

Comment:
Here, this would change requests for the existing code in a way that ?defaultToGlobal=false is always appended. This should not cause issues (e.g. older versions of schema registry that might not know this parameter?).